### PR TITLE
Package Refactor

### PR DESCRIPTION
**Pull request by m0jt4b4**
- [x] changing package from io.in-toto to io.github.in-toto

### CODE REVIEW

File: src/test/java/io/in_toto/lib/AppTest.java [hunk 1]
```diff
@@ -1 +0,0 @@
-package io.in_toto.lib;
```
Comment (SantiagoTorres) [inline src/test/java/io/in_toto/lib/AppTest.java]: Is this not needed? or?

Comment (m0jt4b4) [reply src/test/java/io/in_toto/lib/AppTest.java]: It's [here](https://github.com/in-toto/in-toto-java/blob/f7d950fbb89cda415b6206765a8c4b37bd6244f4/src/test/java/io/github/in_toto/lib/AppTest.java#L1) now

**Comment by SantiagoTorres** [comment]
Updated the README with the changes. it LGTM to me now

File: src/test/java/io/in_toto/lib/AppTest.java (hunk 1 above)
Comment (m0jt4b4) [reply src/test/java/io/in_toto/lib/AppTest.java]: No, it's not. I just modified the same file,
and changed the single line of code from "package io.in_toto.lib;" to "package io.github.in_toto.lib;"
and changed the path from "src/test/java/io/in_toto/lib/AppTest.java" to "src/test/java/io/github/in_toto/lib/AppTest.java"

But, the git diff detected this as new file and considered the old file as deleted.

**Pull request merged**